spec/models/user_spec.rb

```diff
@@ -206,4 +206,39 @@
       end
     end
   end
+
+  describe '.find_or_create_from_gov_one' do
```

Comment:
See if you can improve readability using `with/without/and` rather than `when`

Reply:
[36ee42d](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/917/commits/36ee42d12e27394f9c0a92bce46dc792fed49912)